src/volumetric/volumetric_convex.rs

```diff
@@ -62,6 +62,21 @@ fn tetrahedron_unit_inertia_tensor_wrt_point<P, I>(point: &P, p1: &P, p2: &P, p3
     res
 }
 
+#[allow(non_snake_case)]
+fn triangle_unit_inertia_tensor_wrt_point2D<P>(point: &P, p1: &P, p2: &P, p3: &P) -> <P::Vect as Vect>::Scalar
```

Comment:
I'd prefer not to allow `non_snake_case` as this is non-idiomatic. Instead the "convention" I tend to use is to append the dimension number without the `D`. For example: `triangle_unit_inertia_tensor_wrt_point2`.